front/src/app/models/scan.ts

```diff
@@ -1,8 +1,8 @@
 export interface Scan {
     id: number;
-    tool: string;
-    date: Date;
-    project: string;
+    toolName: string;
+    stringDate: string;
+    project: any;
```

Comment:
`any` jest złym pomysłem. Podajcie konkretny typ, a jak spodziewacie się na przykład stringa albo liczby w danym miejscu, to można użyć unii, przykładowo 
```js
project: string | number;
```